sdks/java/io/mongodb/src/main/java/org/apache/beam/sdk/io/mongodb/MongoDbIO.java

```diff
@@ -191,7 +193,29 @@ private MongoDbIO() {}
       abstract Builder setQueryFn(
           SerializableFunction<MongoCollection<Document>, MongoCursor<Document>> queryBuilder);
 
-      abstract Read build();
```

Comment:
This one was not renamed, it changed because now it has a default implementation (see below in line 208). The default implementation sets a provider given the respective parameters.